notebooks/06.model_serving_feature_lookup.py

```
@@ -0,0 +1,161 @@
+    else:
+        raise e
+
+# COMMAND ----------
+# MAGIC %md
+# MAGIC ### Call the endpoint
+
+# COMMAND ----------
+# Get token and host
+logger.info("Get token and host to test endpoint...")
+
+token = dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().get()
```

Ruff

Undefined name `dbutils`

(F821)